crates/modelardb_compression/src/models/mod.rs

```diff
@@ -348,6 +365,59 @@ mod tests {
         assert!(ErrorBound::try_new(f32::NAN).is_err())
     }
 
+    // is_value_within_error_bound().
```

Comment:
Should be `Tests for is_value_within_error_bound().`.

Reply:
Fixed.